src/detect-transaction-example-numbers.js

```diff
@@ -0,0 +1,70 @@
+// Provides index of requests and responses within given *transition*, sorted by
+// their position in the original API Blueprint document (from first to last).
+//
+//     [
+//         {
+//             'type': 'httpResponse',
+//             'transaction': fury.minim.elements.HttpTransaction()
+//             'position': 85,
+//         },
+//         ...
+//     ]
+//
+// ## Index Entry (object)
+//
+// - type: httpRequest, httpResponse (enum)
+// - position (number) - Position of the first character relevant to
+//   the request (or response) within the original API Blueprint document.
```

Comment:
I think this should be `/* */` or even `/** */`

Reply:
I like `//` even for multiline comments more. We need to fist fight over this.

Reply:
I don't care. I think large multi-line comments which are documentation comments to functions, should be in the jsdoc style, but I'm not going to fight over it until we actually use jsdoc. Well, not jsdoc, but [sphinx-js](https://hacks.mozilla.org/2017/07/introducing-sphinx-js-a-better-way-to-document-large-javascript-projects/).

Reply:
![Eddie](http://replygif.net/i/281.gif)